lib/srv/db/common/permissions/import_rules_test.go

```diff
@@ -404,3 +406,193 @@ func Test_databaseObjectScopeMatch(t *testing.T) {
+				"database_service_name": "service3",
+				"object_kind":           "table",
+				"database":              "db3",
+				"schema":                "schema3",
+				"name":                  "object3",
+			},
+			wantMatch: true,
+		},
+		{
+			name: "add prefix",
+			mapping: &databaseobjectimportrulev1.DatabaseObjectImportRuleMapping{
+				Match: &databaseobjectimportrulev1.DatabaseObjectImportMatch{
+					TableNames: []string{"*"},
+				},
+				AddLabels: map[string]string{
+					"plain_label": "rw",
+					"tag":         "db-{{obj.object_kind}}",
+				},
+			},
+			spec: &dbobjectv1.DatabaseObjectSpec{
+				Database:            "db3",
+				DatabaseServiceName: "service3",
+				Protocol:            "postgres",
+				ObjectKind:          ObjectKindTable,
+				Name:                "object3",
+				Schema:              "schema3",
+			},
+			labels: map[string]string{},
+			wantLabels: map[string]string{
+				"plain_label": "rw",
+				"tag":         "db-table",
+			},
+			wantMatch: true,
+		},
+		{
+			name: "error removes label",
+			mapping: &databaseobjectimportrulev1.DatabaseObjectImportRuleMapping{
+				Match: &databaseobjectimportrulev1.DatabaseObjectImportMatch{
+					TableNames: []string{"*"},
+				},
+				AddLabels: map[string]string{
+					"plain_label": "rw",
+					"tag":         "db-{{obj.invalid}}",
+				},
+			},
+			spec: &dbobjectv1.DatabaseObjectSpec{
+				Database:            "db3",
+				DatabaseServiceName: "service3",
+				Protocol:            "postgres",
+				ObjectKind:          ObjectKindTable,
+				Name:                "object3",
+				Schema:              "schema3",
+			},
+			labels: map[string]string{
+				"tag": "will-be-removed",
+			},
+			wantLabels: map[string]string{
+				"plain_label": "rw",
+			},
+			wantMatch: true,
+		},
+		{
+			name: "multiple values",
+			mapping: &databaseobjectimportrulev1.DatabaseObjectImportRuleMapping{
+				Match: &databaseobjectimportrulev1.DatabaseObjectImportMatch{
+					TableNames: []string{"*"},
+				},
+				AddLabels: map[string]string{
+					"plain_label": "rw",
+					"tag":         "db-{{obj.database}}-{{obj.schema}}-{{obj.name}} ({{obj.object_kind}})",
+				},
+			},
+			spec: &dbobjectv1.DatabaseObjectSpec{
+				Database:            "db3",
+				DatabaseServiceName: "service3",
+				Protocol:            "postgres",
+				ObjectKind:          ObjectKindTable,
+				Name:                "object3",
+				Schema:              "schema3",
+			},
+			labels: map[string]string{
+				"tag": "will-be-removed",
+			},
+			wantLabels: map[string]string{
+				"plain_label": "rw",
+				"tag":         "db-db3-schema3-object3 (table)",
+			},
+			wantMatch: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			labels := maps.Clone(tt.labels)
+			match := applyMappingToObject(tt.mapping, tt.spec, labels)
+			require.Equal(t, tt.wantMatch, match)
+			require.Equal(t, tt.wantLabels, labels)
+		})
+	}
+}
+
+func Test_splitExpressions(t *testing.T) {
+	tests := []struct {
```

Comment:
Can we, please, add more tests to this?

`{{}}` and space before or after `{{ obj.name}}` `{{obj.name }}`

Reply:
I was sure I've added more tests here, sorry for the original omission. In any case this is now removed, as I've exposed some utility functions I'm using for parsing the text instead.